### DISCUSSION TITLE
Building models at panel level other than geo

### DISCUSSION THREAD

**Discussion by Noufa-Mukhtar** [google/meridian#150] [General]
Can we include another panel or subcategory information like geo to run the meridian?

**Comment by alexwakim** [comment]
The main benefit of the geo-hierarchy is that data at the geo-dimension often adds a lot of information, ultimately helping the quality of inference of the causal effect of media (see [Geo-level Bayesian Hierarchical Media Mix Modeling](https://research.google/pubs/geo-level-bayesian-hierarchical-media-mix-modeling/) for more on why). Geo is a good choice for a segmentation dimension because geo-data is often available, media execution often varies across geos, there’s often many geos, and we don’t expect the effect of media to vary wildly across geos.

Other segmentation options might theoretically provide similar advantages. However, in practice, I expect it will be challenging to find a segmentation that is much better than geo. If another segmentation is believed to offer comparable benefits, it must meet the following criteria:

- The segmentation dimension must have mutually exclusive categories
- Input data (media, KPI, and controls) must exist for all categories in the segmentation dimension

Also, note that Media and KPI are automatically scaled by the population size in order to encourage similarity across geo effects (which helps the quality of inference). So there should be a way to measure the size of each category in the segmentation dimension. Although, I suppose you could hack this by setting `population` to 1 for all categories.

**Reply by Noufa-Mukhtar** [reply]
Hey Alex, I agree with the geo segmentation and I would like to have that for my models but I have another segmentation which is a campaign. I have several campaigns and would like to know the effectiveness of all media channels across these campaigns. Is there any way we can include this information in Meridian currently?

**Reply by alexwakim** [reply]
Geo-level data is often available because it’s often unambiguous how much of the KPI can be attributed to the geo. Say the KPI is “units sold”. It’s unambiguous which units were sold in which geo. The same cannot be said if the hierarchical segmentation is campaign. It’s often very ambiguous which campaign a unit sold is “in”.

**Reply by Noufa-Mukhtar** [reply]
That is true but it is also the same case of ambiguity for which media channel led to that sale. For the campaign segmentation, does Merdian have a proposed or suggested solution as of now? Also, how can we look at the output values at the geo level?

**Reply by alexwakim** [reply]
You need to provide Meridian with data at the geo-level to use the geo-hierarchy. If you want to use campaign instead of geo as a segmentation dimension, you'll need to provide data at the campaign-level. It's not clear how to provide the KPI at the campaign level.

Many of the methods in the Analyzer class can provide geo-level results.

Our advice on campaign level can be found here: https://developers.google.com/meridian/docs/advanced-modeling/additional-considerations#campaign-level-data
